### PR TITLE
Feat/add person class

### PR DESCRIPTION
**Pull request by MartinHermes86**

### CODE REVIEW

File: src/main/java/co/github/martinhermes86/Person.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package co.github.martinhermes86;
+
+import java.util.Objects;
+
+public class Person {
+    private String name;
+    private int age;
+
```
Comment (JanaS87) [inline src/main/java/co/github/martinhermes86/Person.java:9]: Looks nice maybe you could add an address for the person?
```suggestion
private String address;
```

**Review by JanaS87 — COMMENTED** [review]
Hey, did some suggestions.
Looks good to me!

File: src/main/java/co/github/martinhermes86/Person.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
+package co.github.martinhermes86;
+
+import java.util.Objects;
+
+public class Person {
+    private String name;
+    private int age;
```
Comment (ReebalSami) [inline src/main/java/co/github/martinhermes86/Person.java:7]: add hight properties?!

**Review by ReebalSami — CHANGES_REQUESTED** [review]

File: src/main/java/co/github/martinhermes86/Person.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
+package co.github.martinhermes86;
+
+import java.util.Objects;
+
+public class Person {
+    private String name;
```
Comment (betulEZ) [inline src/main/java/co/github/martinhermes86/Person.java:6]: add new attribute

**Review by betulEZ — CHANGES_REQUESTED** [review]
add

**Review by ReebalSami — APPROVED** [review]

**Review by JanaS87 — APPROVED** [review]
lgtm!

**Review by betulEZ — APPROVED** [review]

**Pull request merged**